packages/radix-vue/src/shared/types.ts

```diff
@@ -45,6 +45,5 @@ type ScrollBodyOption = {
 
 type AcceptableValue = string | number | boolean | Record<string, any>
 type ArrayOrWrapped<T> = T extends any[] ? T : Array<T>
- type StringOrNumber = string | number
```

Comment:
I think having this type named as `StringOrNumber` is much better than `ModelValue`. `ModelValue` naming can be ambiguous and if more component using this name it's gonna cause some confusion. 

Let's stick it back to `StringOrNumber` please 😁

Reply:
Why would it be confusing to name something more closely resembling its purpose, especially knowing the `ModelValue` type is placed inside of the component it is closely related to (unlike generic `StringOrNumber`)? Two reasons I dislike this abstraction:

- reduces cognitive load when mentioned in the docs (as a user, I'd much rather see a prop is of type `string | number` than `StringOrNumber`)
- why should we abstract something so short and obvious? If the type ever changes, so does the name `StringOrNumber`. I can't see the purpose of having it at all.

Nevertheless, let me know if the above makes sense at all - I'm happy to pick whichever option we decide on 😄